tmt/steps/provision/__init__.py

```diff
@@ -356,6 +356,7 @@ class GuestFacts(SerializableContainer):
     is_ostree: Optional[bool] = None
     is_toolbox: Optional[bool] = None
     toolbox_container_name: Optional[str] = None
+    container: Optional[str] = None
```

Comment:
At first I thought "`is_container` would be better", but then I realized it's actually a container *name*. I'd recommend changing the attribute then, to make it clear what it carries, `container_name`.

And we already have `toolbox_container_name`...

Reply:
Good point, I actually started with is_container and eventually get to container (name) but I am a little skeptical about the value of keeping the name (it does not really matter  and I'll get back to is_container again. If there will ever be a use case for actual container implementation, it can be added eventually.

Reply:
Resolved in [7a3a09b](https://github.com/teemtee/tmt/pull/3344/commits/7a3a09bd9f5c367a92cbd4d0e895c6572f8d4aaf).